tests/test_command_reporter.py

```diff
@@ -238,11 +241,11 @@ def test_should_suppress_stdout_and_stderr_based_on_command_line(ClientMock, sys
         time_spent=mock.ANY,
         data=mock.ANY,
         extra={
-            'command': mock.ANY,
-            'exit_status': mock.ANY,
-            "last_lines_stdout": expected_stdout,
-            "last_lines_stderr": expected_stderr,
-    })
+            'command':mock.ANY,
```

Comment:
Could you revert all stylistic changes, please? they are not related to the pull request.